src/Protocol/ActivityPub/Processor.php

```diff
@@ -688,12 +692,13 @@ private static function postMail($activity, $item)
 	/**
 	 * Fetches missing posts
 	 *
-	 * @param string $url message URL
-	 * @param array $child activity array with the child of this message
+	 * @param string $url       message URL
+	 * @param array  $child     activity array with the child of this message
+	 * @param bool   $relaymode Posts arrived via relay
 	 * @return string fetched message URL
 	 * @throws \Friendica\Network\HTTPException\InternalServerErrorException
 	 */
-	public static function fetchMissingActivity(string $url, array $child = [])
+	public static function fetchMissingActivity(string $url, array $child = [], bool $relaymode = false)
```

Comment:
Can we have the domain of the relay passed to this method?

Reply:
Technically this would be possible. I would add/change it if needed.

Reply:
The more info we get, the better for later troubleshooting/statistics. This could allow node admins to finely tune their relay subscriptions.